resolwe_bio/processes/import_data/bwamem2_index_files.py

```diff
@@ -0,0 +1,75 @@
+"""Upload index files for BWA-MEM2 aligner"""
+import gzip
+import shutil
+from pathlib import Path
+
+from resolwe.process import (
+    Cmd,
+    DataField,
+    DirField,
+    FileField,
+    Process,
+    SchedulingClass,
+    StringField,
+)
+
+
+class ImportBWA2Index(Process):
+    """Import BWA-MEM2 index files."""
+
+    slug = "upload-bwamem2-index"
+    name = "BWA-MEM2 index files"
+    process_type = "data:index:bwamem2"
+    version = "1.0.0"
+    category = "Import"
+    scheduling_class = SchedulingClass.BATCH
+    requirements = {
+        "expression-engine": "jinja",
+        "executor": {
+            "docker": {"image": "public.ecr.aws/genialis/resolwebio/dnaseq:6.1.0"},
+        },
+    }
+    data_name = '{{ ref_seq.fasta.file|basename|default("?") }}'
+    version = "1.0.0"
+
+    class Input:
+        """Input fields to process Import BWA-MEM2 index."""
+
+        ref_seq = DataField(
```

Comment:
It's best if also the FASTA on input `ref_seq` is to be uploaded (`FileField`) in this process. With this, you'll also need to modify the `data_name` to be `'{{ ref_seq.file|default("?") }}'`